charter/index.html

```diff
@@ -267,7 +270,7 @@ <h3>
         <section id="dependencies">
           <h3>Dependencies</h3>
 
-          <p>The proposed adopted draft for the Solid Protocol has a number of normative dependencies with a maturity level insufficient for a recommendation (Community Group reports, Editor's Drafts). The Working Group will need to decide how to deal with these dependencies in order to allow the Solid Protocol v1.0 to progress on the Recommendation track. This may include downgrading those dependencies to non-normative references, specifying the required parts in the Solid Protocol itself, or relying on other specifications to fulfil the same purpose.</p>
+          <p>The draft of the PUMPKIN Protocol proposed for adoption has a number of normative dependencies with maturity levels insufficient for a recommendation (e.g., Community Group reports, Editor's Drafts). The Working Group will need to decide how to deal with these dependencies in order to allow the PUMPKIN Protocol v1.0 to progress on the Recommendation track. This may include downgrading some of these dependencies to non-normative references; specifying the required parts in the PUMPKIN Protocol itself; and/or relying on other specifications to fulfill the same purpose.</p>
```

Comment:
```suggestion
          <p>The draft of the PUMPKIN Protocol proposed for adoption has a number of normative dependencies with maturity levels insufficient for a recommendation (e.g., Community Group reports, Editor's Drafts). To allow the PUMPKIN Protocol v1.0 to progress on the Recommendation track, the Working Group will need to decide how to deal with these dependencies. This may include downgrading some of these dependencies to non-normative references; specifying the required parts in the PUMPKIN Protocol itself; and/or relying on other specifications to fulfill the same purpose.</p>
```
```suggestion
          <p>The draft of the PUMPKIN Protocol proposed for adoption has a number of normative dependencies with maturity levels insufficient for a recommendation (e.g., Community Group reports, Editor's Drafts). The Working Group will need to decide how to deal with these dependencies in order to allow the PUMPKIN Protocol v1.0 to progress on the Recommendation track. This may include downgrading some of these dependencies to non-normative references; specifying the required parts in the PUMPKIN Protocol itself; and/or relying on other specifications to fulfill the same purpose.</p>
```